### ISSUE TITLE
Action set_value throws an exception of ..

### ISSUE THREAD

**Issue by minimul** [marcoroth/turbo_power-rails#11]
`ArgumentError (You invoked render but did not give any of :partial, :template, :inline, :file, :plain, :html or :body option.):`, while the same usage with `set_attribute` does not.

`turbo_stream.set_value('#order_form_manual_time', '')` **=> throws exception**

`turbo_stream.set_attribute('#order_form_manual_time', 'value', '')` **=> works**

I looked briefly at `https://github.com/marcoroth/turbo_power-rails/blob/main/lib/turbo_power/stream_helper.rb` and everything seems in order.

I can do more investigating and attempt to produce a failing test later but wanted to get this potential issue recorded.

**Comment by marcoroth** [comment]
I think I know what's up, but not sure how I can fix it.

I might just remove the block rendering syntax support for now.

**Comment by minimul** [comment]
You know actually `set_attribute` is not working either for setting the value:

```
<%= turbo_stream.set_attribute('#order_form_manual_time', 'value', '') %><!-- Doesn't work -->
<%= turbo_stream.invoke("value =", args: [""], selector: '#order_form_manual_time') %><!-- Works -->
```

**Comment by minimul** [comment]
Or should I say that the the form field `#order_form_manual_time`, which is a `input[type=text]` element, doesn't get 'blanked' so that when the entire form is then resubmitted after the `set_attribute` stream it is **not** empty on the form submission. The `.invoke` will indeed blank out the field in question on submission.

**Comment by marcoroth** [comment]
@minimul okay, I think #12 should have solved this issue. I'll cut a release shortly. 

Otherwise you can point your `turbo_power` gem to this in the meantime:
```ruby
gem "turbo_power", github: "marcoroth/turbo_power-rails", branch: "main"
```

**Comment by minimul** [comment]
I can confirm both `set_value` and `set_attribute(target, 'value', ...` are working off of `main`. Thanks!

**Comment by marcoroth** [comment]
Awesome, it's now released as `v0.2.0`
